client/src/containers/mealDetails.js

```diff
@@ -2,6 +2,8 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { fetchMealDetail } from '../actions/index';
 import { Link } from 'react-router-dom';
+import {Card, CardActions, CardHeader, CardMedia, CardTitle, CardText} from 'material-ui/Card';
```

Comment:
Add spaces to the beginning and end when you destructure here, i.e., `{ Card, ..., CardText }`